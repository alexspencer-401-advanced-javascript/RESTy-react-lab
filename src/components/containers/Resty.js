import React from 'react';
import Header from '../Header';
import Footer from '../Footer';
import History from '../History';
import Form from '../Form';
import Results from '../Results';

export default function App() {
  return (
    <>
      <Header />
      <History />
      <Form />
      <Results />
      <Footer />
    </>
  );
}
