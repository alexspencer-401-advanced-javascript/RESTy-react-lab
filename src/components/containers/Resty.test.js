import React from 'react';
import { shallow } from 'enzyme';
import Resty from './Resty';

describe('Resty component', () => {
  it('renders Resty', () => {
    const wrapper = shallow(<Resty />);
    expect(wrapper).toMatchSnapshot();
  });
});
