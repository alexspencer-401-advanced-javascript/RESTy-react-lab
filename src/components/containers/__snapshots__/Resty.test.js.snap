// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Resty component renders Resty 1`] = `
<Fragment>
  <Header />
  <Form />
  <Results />
  <Footer />
</Fragment>
`;
